polymode-core.el

```diff
@@ -951,6 +951,9 @@ Parents' hooks are run first."
     selective-display
     text-scale-mode
     text-scale-mode-amount
+    ;; transient-mark-mode stores here the state of selection
+    ;; when the shift-select-mode is enabled
+    transient-mark-mode
```

Comment:
This is treated in [here](https://github.com/polymode/polymode/pull/254/files#diff-a0abe25df0bc846c9f176364ba3b5f16R1055). There is a fixme comment there which I didn't have time to fix yet. Could you please check if with your change that mark-reactivation is still needed?

Reply:
It seems it's still needed. Without it buffers don't synchronize selections - the old one will remain active mark and the new one won't take it. The reason transient-mark-mode has to be transferred to a new buffer is that it stores state of selection made with shift, i.e. without it new buffer will remove selection even if user still holds shift.